confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java

```diff
@@ -839,10 +841,27 @@ public UDFManager getUDFManager() {
   }
 
   @Override
-  public DataSet showRegion(GetRegionInfoListPlan getRegionInfoListPlan) {
+  public RegionInfoListResp showRegion(GetRegionInfoListPlan getRegionInfoListPlan) {
     TSStatus status = confirmLeader();
     if (status.getCode() == TSStatusCode.SUCCESS_STATUS.getStatusCode()) {
-      return partitionManager.getRegionInfoList(getRegionInfoListPlan);
+      RegionInfoListResp regionInfoListResp =
+          (RegionInfoListResp) partitionManager.getRegionInfoList(getRegionInfoListPlan);
+      regionInfoListResp
+          .getRegionInfoList()
+          .forEach(
+              regionInfo -> {
+                Map<TConsensusGroupId, Integer> allLeadership = loadManager.getAllLeadership();
```

Comment:
Please update the getAllLeadership() interface to adapt MultiLeader consensus protocol~

Reply:
fixed~